controllers/src/main/scala/cromwell/pipeline/controller/ControllerModule.scala

```diff
@@ -18,4 +18,7 @@ class ControllerModule(serviceModule: ServiceModule, authConfig: AuthConfig)(
   lazy val projectFileController: ProjectFileController = new ProjectFileController(serviceModule.projectFileService)
   lazy val configurationController = new ProjectConfigurationController(serviceModule.configurationService)
   lazy val runController: RunController = new RunController(serviceModule.runService)
+  lazy val projectSearchController: ProjectSearchController = new ProjectSearchController(
+    serviceModule.projectSearchService
+  )
```

Comment:
[minor] can we make it prettier please?
```suggestion
  lazy val projectSearchController: ProjectSearchController =
    new ProjectSearchController(serviceModule.projectSearchService)
```

Reply:
Done